tests/test_liste_simple.py

```diff
@@ -120,8 +120,22 @@ def test_setitem():
 
 def test_eq():
     # test si l'opérateur == fonctionne correctement sur une liste chaînée simple
-    pass
+    l = Liste()
+    l2 = Liste()
+    for i in range(5):
+        l.inserer_queue(i)
+        l2.inserer_queue(i)
+    assert 1 == l2
+    l2[0] = 56
+    assert (l == l2) == False
```

Comment:
maladroit: `assert not (l == l2)` ou `assert l != l2`

Reply:
Fix